lib/utils/AbstractQueryConfiguration.dart

```diff
@@ -38,7 +48,16 @@ abstract class AbstractQueryConfiguration {
         (field) => field == ProductField.ALL,
       );
       if (!ignoreFieldsFilter) {
-        final fieldsStrings = convertFieldsToStrings(fields!, language);
+        final languages = secondaryLanguages.toList();
+        if (language != null && !languages.contains(language)) {
+          languages.add(language!);
```

Comment:
If both language and scondaryLanguages are provided, language should be added at the top of the list I think.

Reply:
Or on second thought, I think we should disallow it and throw an error.

Reply:
Are you sure?
The 2 fields regulate different behavior - the `language` field gives a value to the `lc` query-param, and `languages`/`secondaryLanguages` give values to lang-postfix query params (e.g. `product_name_en`).
`lc` and postfixes have different behaviors: https://github.com/openfoodfacts/openfoodfacts-dart/pull/185#issuecomment-882654523